app/javascript/.storybook/preview.js

```diff
@@ -86,27 +32,8 @@ addParameters({
   },
 });
 
-export const Foo = () => <Component />;
```

Comment:
Some legacy stuff that wasn't used anywhere and in fact was throwing errors in console...